server/server.js

```diff
@@ -36,13 +36,38 @@ sockets.init(server);
 server.listen(PORT, () => {
   console.log('Example app listening at', server.address().port);
 });
+const HEADER = 'MHP DAShboard';
+const sidebar = [
+  { file: 'index', title: 'Dashboard' },
+  { file: 'files', title: 'Download Files' },
```

Comment:
Let's change this to be `{ file: 'download-files', title: 'Download Files' },` instead so we don't get a clash in endpoints?

Reply:
ohhh i get it now! alright i'll do that :)